### ISSUE TITLE
[Help needed] Hosting for Github Awards

### ISSUE THREAD

**Issue by vdaubry** [vdaubry/github-awards#96]
### TLDR;
**We need to find a sponsor to buy a server with at least 8GB of RAM** 
It's a matter of time before we totally run out of memory to hold rankings, and i'll be forced to shutdown the service :finnadie:

We need either :

* A hosting provider willing to host the project
* Any sponsor willing to donate enough to get a EC2 m3.large instance : $713 for 1 year

In return the sponsor would be able to display some sort of "sponsored by" message on the website.
The website is visited by more than 20k unique developers every month.

### Longer version :
Hi everyone,

Over the last few weeks there have been recurring problems with rankings (#81, #73, #89, #94, etc)

The underlying issue is that rankings data are growing over time and the server hosting Github Awards doesn't have enough RAM to support this growth.

The current server is a Rackspace cloud instance with 2GB of RAM.

Github Awards is a free project without any sponsoring at the moment. In order to keep the project growing i need to find a host willing to provide a server instance with at least 4GB of RAM.

### What's next ?

Starting today i'm going to contact all major hosting provider, hoping to find someone willing to provide some hosting for the project.
i'm also going to contact companys with product targeting developers

### How can you help ?

If you know some company that could be interested in reaching a developer audience by sponsoring Github Awards, tell them to send me a message : vdaubry at gmail dot com

I'm also looking for help to tune Redis and Postgres to improve the behavior when the system memory is exhausted.

For information :
Ranks are computed using Redis, wich is why all ranks needs to be kept in memory.

The site is currently hosted for free on Rackspace thanks to their developper credits program.
